python/lsst/pipe/analysis/utils.py

```diff
@@ -19,6 +19,8 @@
 import lsst.afw.table as afwTable
 import lsst.pex.config as pexConfig
 
+from lsst.qa.explorer.parquetTable import ParquetTable
+
```

Comment:
Need to add this to `ups/pipe_analysis.table`.  I would also suggest it be made a {{setupOptional}} and only trying to import it if `doWriteParquetTables=True`

Reply:
I'd be OK with making this optional, but I would counter-advocate that it might be a better idea to make it required (and to get rid of the option of `doWriteParquetTables` altogether).  I guess in the future we actually want `ParquetTable` to break out of `qa_explorer` anyway, right @yalsayyad?

Reply:
It is really nice to have the option (at least for now) to not write parquet files when testing plot output.  If/when we "break-out", the option can be removed then.

Reply:
Yeah, I'm OK with that.  I just put the try import block back into the `writeParquet` function.